packages/components/src/components/wizard/wizard.tsx

```diff
@@ -1,42 +1,57 @@
-import PropTypes from 'prop-types';
+import PropTypes, { object } from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
-import Step from './wizard-step.jsx';
+import Step from './wizard-step';
+
+type TWizard = {
+    className?: string;
+    initial_step: number;
+    onStepChange: (prop: { [key: string]: number }) => void;
+    nav: React.ReactNode;
+    selected_step_ref: () => any | React.MutableRefObject<HTMLElement>;
+};
 
-const Wizard = ({ children, className, initial_step, onStepChange, nav, selected_step_ref }) => {
+const Wizard = ({
+    children = [],
+    className,
+    initial_step = 1,
+    onStepChange = () => undefined,
+    nav = null,
+    selected_step_ref,
+}: React.PropsWithChildren<TWizard>) => {
     const [active_step, setActiveStep] = React.useState(0);
 
```

Comment:
I believe we should also add types for react hooks

Reply:
For `useState` we don't need since TS can infer the type here. but for `getSteps` yes

Reply:
Do we need to provide types explicitly to `const`?

Reply:
I'm not sure but as I checked it we don't need to type `getSteps`.